system-addon/content-src/components/ContextMenu/ContextMenu.jsx

```diff
@@ -0,0 +1,42 @@
+const React = require("react");
+
+const ContextMenu = React.createClass({
```

Comment:
can you use `class ContextMenu extends React.Component` since it is slightly more performant?